Tests/UnitTests/Plugins/Geant4/Geant4ConvertersTests.cpp

```diff
@@ -356,7 +360,8 @@ BOOST_AUTO_TEST_CASE(Geant4LineVPhysConversion) {
   Acts::ActsScalar thickness = 400.;
 
   G4Tubs* g4Tube = new G4Tubs("Line", innerRadius, outerRadius, 0.5 * thickness,
-                              -M_PI * CLHEP::radian, 2 * M_PI * CLHEP::radian);
+                              -std::numbers::pi * CLHEP::radian,
```

Comment:
I forgot how G3Tubs goes, but I guess it's starting value and opening angle otherwise this looks weird.